app/main.py

```diff
@@ -14,4 +14,5 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    res = phrase.lower().count(letter.lower())
+    return res
```

Comment:
It is better to use full names for variables, result - not res

Reply:
> It is better to use full names for variables, result - not res

remade